src/main/java/com/thealgorithms/geometry/GrahamScan.java

```diff
@@ -20,17 +19,12 @@ public class GrahamScan {
     private final Stack<Point> hull = new Stack<>();
 
     public GrahamScan(Point[] points) {
-
-        /*
-         * pre-process the points by sorting them with respect to the bottom-most point, then we'll
-         * push the first point in the array to be our first extreme point.
-         */
+        // Pre-process points: sort by y-coordinate, then by polar order with respect to the first point
         Arrays.sort(points);
         Arrays.sort(points, 1, points.length, points[0].polarOrder());
         hull.push(points[0]);
 
-        // find index of first point not equal to a[0] (indexPoint1) and the first point that's not
-        // collinear with either (indexPoint2).
+        // Find the first point not equal to points[0] and the first point not collinear with the previous points
         int indexPoint1;
```

Comment:
This name does not provide any information about this variable. 

Maybe this one firstNonEqualIndex will give more understanding and clearity. 

And firstNonCollinearIndex instead of indexPoint2.

Reply:
added 👍